examples/disaster-recovery/README.md

```diff
@@ -0,0 +1,49 @@
+# Coordinate disaster recovery with Secret Manager
+
+## Background
+
+This document assumes you are already using the following strategy for
+detecting and triggering failovers:
+1. Using an independent service to detect when the primary is down
+2. Trigger a promotion of an existing read replica to become a primary
+3. Update a Secret Manager secret with the name of the current primary
+
+## Restart Auth proxy when secret changes
+
+This option uses a wrapper script around the Cloud SQL Auth proxy to detect
+when the secret has changed, and restart the proxy with the new value. This
+could be done in many languages, but here’s an example using bash:
+
+```sh
+#! /bin/bash
+
+SECRET_ID="my-secret-id" # TODO(developer): replace this value
+PORT=5432
+
+# Get the latest version of the secret and start the proxy
+INSTANCE=$(gcloud secrets versions access "latest" --secret="$SECRET_ID")
+cloud_sql_proxy -instances="$INSTANCE"=tcp:"$PORT" &
+PID=$!
+
+# Every 5s, get the latest version of the secret. If it's changed, restart the
+# proxy with the new value.
+while true; do
+    sleep 5
```

Comment:
nit: Should sleep be configurable? Some users might want to adjust

Reply:
I'll make it so 👍 I believe it helps having all the configurable params up there at the top 😊